crates/gitbutler-stack/src/stack.rs

```diff
@@ -655,6 +658,33 @@ impl Stack {
         Ok(())
     }
 
+    /// Sets the forge identifiers for a given series/branch. Existing values are overwritten.
+    ///
+    /// # Errors
+    /// If the series does not exist, this method will return an error.
+    /// If the stack has not been initialized, this method will return an error.
+    pub fn set_forge_ids(
+        &mut self,
+        ctx: &CommandContext,
+        series_name: &str,
+        new_forge_ids: Vec<ForgeIdentifier>,
+    ) -> Result<()> {
+        if !self.initialized() {
+            return Err(anyhow!("Stack has not been initialized"));
+        }
```

Comment:
We have these three lines in 9 other places, could we maybe make something like `self.assertInitialized()?;` that either returns this error or just Ok's?

Reply:
Yeah good idea. the `initialized()` method itself could do that. This code is probably ready for a light refactor now, since i have only been adding to it for a couple of weeks